src/bidiMapper/modules/browser/UserContextStorage.ts

```diff
@@ -46,6 +49,14 @@ export class UserContextStorage {
     ];
   }
 
+  getConfig(userContextId: string): UserContextConfig {
```

Comment:
When do we delete thease object, while they are small now, I would like for them to have a clean up.

Reply:
I don't think we have a mechanism to detect browser context destroyed.